Resources/Prototypes/Alerts/alerts.yml

```diff
@@ -0,0 +1,205 @@
+  description: "The air around you is [color=red]hazardously thin[/color]. A [color=green]space suit[/color] would protect you."
+
+- type: alert
+  id: highpressure
+  category: pressure
+  icon: /Textures/Interface/StatusEffects/Pressure/highpressure.png
+  maxSeverity: 2
+  name: "[color=red]High Pressure[/color]"
+  description: "The air around you is [color=red]hazardously thick[/color]. A [color=green]fire suit[/color] would protect you."
+
+- type: alert
+  id: fire
+  icon: /Textures/Interface/StatusEffects/Fire/fire.png
+  name: "[color=red]On Fire[/color]"
+  description: "You're [color=red]on fire[/color]. Click the alert to stop, drop and roll to put the fire out or move to a vacuum area."
+
+- type: alert
+  id: cold
+  category: temperature
+  icon: /Textures/Interface/StatusEffects/Temperature/cold.png
+  maxSeverity: 3
+  name: "[color=cyan]Too Cold[/color]"
+  description: "You're [color=cyan]freezing cold![/color] Get somewhere warmer and take off any insulating clothing like a space suit."
+
+- type: alert
+  id: hot
+  category: temperature
+  icon: /Textures/Interface/StatusEffects/Temperature/hot.png
+  maxSeverity: 3
+  name: "[color=red]Too Hot[/color]"
+  description: "It's [color=red]too hot![/color] Flee to space or at least away from the flames. Standing on weeds will heal you."
+
+- type: alert
+  id: weightless
+  icon: /Textures/Interface/StatusEffects/Weightless/weightless.png
+  name: Weightless
+  description: >-
+    Gravity has ceased affecting you, and you're floating around aimlessly. You'll need something large and heavy, like a
+    wall or lattice, to push yourself off if you want to move. A jetpack would enable free range of motion. A pair of
+    magboots would let you walk around normally on the floor. Barring those, you can throw things, use a fire extinguisher,
+    or shoot a gun to move around via Newton's 3rd Law of Motion.
+
+- type: alert
+  id: stun
+  icon: /Textures/Objects/Weapons/Melee/stunbaton.rsi/stunbaton_off.png
+  name: "[color=yellow]Stunned[/color]"
+  description: "You're [color=yellow]stunned[/color]! Wait for it to wear off."
+
+- type: alert
+  id: handcuffed
+  icon: /Textures/Interface/StatusEffects/Handcuffed/Handcuffed.png
+  name: "[color=yellow]Handcuffed[/color]"
+  description: "You're [color=yellow]handcuffed[/color] and can't act. If anyone drags you, you won't be able to move.."
+
+- type: alert
+  id: buckled
+  category: buckled
+  icon: /Textures/Interface/StatusEffects/Buckle/buckled.png
+  name: "[color=yellow]Buckled[/color]"
+  description: "You've been [color=yellow]buckled[/color] to something. Click the alert to unbuckle unless you're [color=yellow]handcuffed.[/color]"
+
+- type: alert
+  id: humancrit
+  category: health
+  icon: /Textures/Interface/StatusEffects/Human/humancrit-0.png
+  name: "[color=red]Critical Condition[/color]"
+  description: "You're severely injured and unconscious."
+
+- type: alert
+  id: humandead
```

Comment:
Shouldn't those be camelCase?

Reply:
Let me quote the official naming guidelines for YAML:
>`PascalCase` is used for IDs and component names. Everything else uses `camelCase`.